.github/workflows/retest.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
     steps:
       # path to the retest action
       # yamllint disable-line rule:line-length
-      - uses: ceph/ceph-csi/actions/retest@28dc64dcae3cec8d11d84bdf525bda0ef757c688  # devel
+      - uses: ceph/ceph-csi/actions/retest@961c0a8d85035a5ee4854e3960ce1b8df8b90166  # devel
```

Comment:
This is our own repository, it should not include the commit-id, the branch name (`devel`) is more appropriate.

@Nikhil-Ladha do you know how to annotate/comment this so that the security scanner accepts the branch?

Reply:
> This is our own repository, it should not include the commit-id, the branch name (`devel`) is more appropriate.
> 
> @Nikhil-Ladha do you know how to annotate/comment this so that the security scanner accepts the branch?

Looking at the current annotation method, it seems like we can't omit a single action it has to be the whole check. Ref: https://github.com/ossf/scorecard/blob/main/config/README.md

Reply:
> or, maybe we can use it like [private actions](https://docs.github.com/en/actions/sharing-automations/creating-actions/creating-a-docker-container-action?learn=create_actions&learnProduct=actions#example-using-a-private-action):
> 
> ```yaml
> - uses: ./actions/retest
> ```
> 
> This points to the local directory in the repository that contains the action. The repository needs to be checked out first, I guess.

This seems like the best possible way for this case, I will send a PR to update the action to use the local directory.

Reply:
Here's the PR: #4879